libs/pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[build-system]
+requires = [
+    "setuptools>=61.0",
+    "setuptools-scm"
+]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "core"
```

Comment:
if too compolicated ,we can remove this pyproject.toml and only keeps at the root of project repo.